lib/resources/port.rb

```diff
@@ -34,8 +34,14 @@ def initialize(ip = nil, port) # rubocop:disable OptionalArguments
     case inspec.os[:family]
     when 'ubuntu', 'debian', 'redhat', 'fedora', 'centos', 'arch', 'wrlinux'
       @port_manager = LinuxPorts.new(inspec)
-    when 'darwin'
-      @port_manager = DarwinPorts.new(inspec)
+    when 'darwin', 'aix'
+      # AIX: see http://www.ibm.com/developerworks/aix/library/au-lsof.html#resources
+      #      and https://www-01.ibm.com/marketing/iwm/iwm/web/reg/pick.do?source=aixbp
```

Comment:
there's a link in the code to where lsof can be downloaded from.  I know it is not standard AIX per se, but honestly there's no good way of doing what we're trying to do here on AIX without it.